### ISSUE TITLE
Error:Failed to resolve: rxbinding-root:rxbinding:unspecified 

### ISSUE THREAD

**Issue by leeyc09** [JakeWharton/RxBinding#95]
When I Setting at Android Studio.
```
dependencies {
 ...
   'com.jakewharton.rxbinding:rxbinding:0.1.0'
   'com.jakewharton.rxbinding:rxbinding-support-v4:0.1.0'
}
```

Error occur. right this.
```
Error:Failed to resolve: rxbinding-root:rxbinding:unspecified
```

how to fix it?

**Comment by JakeWharton** [comment]
You can either exclude the transitive dependency from 'support-v4' or use the latest SNAPSHOT until the next release. This was fixed by 10357c1c37a40ee5f59a203c2a4e5a50ff851e0f

**Comment by leeyc09** [comment]
Thanks
